### PR TITLE
Governable Lookback Window (CIP-0021)

### PR DESCRIPTION
**Pull request by andres-dg**

### CODE REVIEW

File: CIPs/0021.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# CIP [0021]: Governable lookback window
+
+- Date: 2020-09-25
+- Author: @andres-dg
```
Comment (YazzyYaz) [inline CIPs/0021.md:4]: hi @andres-dg  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo after the other action items are completed.

**Review by YazzyYaz — CHANGES_REQUESTED** [review]

File: CIPs/0021.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+cip: <to be assigned>
+title: Governable lookback window
+author: Andres di Govanni (@andres-dg)
+discussions-to: <to be assigned>
```
Comment (YazzyYaz) [inline CIPs/0021.md:5]: please create a github issue for your proposal called "Discussions for Governable Lookback Window"

**Review by YazzyYaz — COMMENTED** [review]
I'd also add to rename the file to cip-0021.md

Once you do that and add the link for discussions, I'll approve the PR for a Draft

**Review by prestwich — APPROVED** [review]

**Comment by prestwich** [comment]
This will need more specification after merging as a draft

File: CIPs/0021.md (hunk 2 above)
Comment (YazzyYaz) [reply CIPs/0021.md:5]: @andres-dg just made it for you https://github.com/celo-org/celo-proposals/issues/86

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**